### DISCUSSION TITLE
`bom-ref` propertyused in PowerShell.

### DISCUSSION THREAD

**Discussion by sergiomarotco** [CycloneDX/specification#487] [Q&A]
## Describe the feature
Object bom-ref out of specification camelCase. In PowerShell i have «Unexpected Token» error:
![image](https://github.com/CycloneDX/specification/assets/29877074/957161d1-17a1-4858-8589-2622defcb537)

## Possible solutions

bom-ref rename to bomRef

**Comment by jkowalleck** [comment]
@sergiomarotco could you provide a complete reproducible example? 

I could imagine this is a general "you are using it wrong" thing.
Have you tried `$vuln['bom-ref']` or whatever applies to your environment? I mean, in general, `-` is a mathematical operator, and because of this, you might use the general attribute operator `[]`, instead of the scope operation `.` ...
